examples/run_examples.sh

```diff
@@ -3,36 +3,48 @@
 # set -e
 set -o nounset
 
+rm -f *.vtu *.pvtu
+
 origin=$(pwd)
 examples_dir=${1-$origin}
 declare -i exitcode=0
-echo "Running examples in $examples_dir ..."
+echo "*** Running examples in $examples_dir ..."
 failed_examples=""
 for example in $examples_dir/*.py
 do
-    if [[ "$example" == *"-mpi.py" ]]
+    if [[ "$example" == *"-lazy-mpi.py" ]]
     then
-        echo "Running parallel example: $example"        
+        echo "*** Running parallel lazy example (1 rank): $example"
+        mpiexec -n 1 python -m mpi4py ${example} --lazy
+        rm -rf *vtu *sqlite *pkl *-journal restart_data
```

Comment:
Maybe add *.pvtu?

Reply:
*vtu already gets the .pvtus!  :)

Reply:
It's late, sorry 😅

Reply:
No worries! I realize unix is challenging late at night.